### PR TITLE
53520 : improve methode accountingMultipleMoves

### PR DESCRIPTION
**Pull request by alg-axelor**

### CODE REVIEW

File: axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java [hunk 1]
```diff
@@ -145,30 +145,15 @@ public void generateReverse(ActionRequest request, ActionResponse response) {
 
   @SuppressWarnings("unchecked")
   public void validateMultipleMoves(ActionRequest request, ActionResponse response) {
-    List<Long> moveIds = (List<Long>) request.getContext().get("_ids");
+    List<Integer> moveIds = (List<Integer>) request.getContext().get("_ids");
```
Comment (jmi-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java:148]: Why did you change it from List<Long> to List<Integer>?

**Review by jmi-axelor — CHANGES_REQUESTED** [review]

File: axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java (hunk 1 above)
Comment (alg-axelor) [reply axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java:148]: Because the list comes from the context, and somehow, if I keep the List<Long>, it throws a cast exception, and in the debugger, the List<Long> contains integers. This solution is used in a lot of methods in the app.

**Review by jmi-axelor — APPROVED** [review]
